src/App/ExpressionParser.l

```diff
@@ -170,6 +170,11 @@ EXPO     [eE][-+]?[0-9]+
 "l"                          COUNTCHARS; yylval.quantity.scaler  = Quantity::Liter;               yylval.quantity.unitStr = yytext; return UNIT; // Liter      dm^3
 "ml"                         COUNTCHARS; yylval.quantity.scaler  = Quantity::MilliLiter;          yylval.quantity.unitStr = yytext; return UNIT; // milli Liter
 
+"Hz"                         COUNTCHARS; yylval.quantity.scaler  = Quantity::Hertz;               yylval.quantity.unitStr = yytext; return UNIT; // Hertz
+"kHz"                        COUNTCHARS; yylval.quantity.scaler  = Quantity::KiloHertz;           yylval.quantity.unitStr = yytext; return UNIT; // kilo Hertz
+"MHz"                        COUNTCHARS; yylval.quantity.scaler  = Quantity::MegaHertz;           yylval.quantity.unitStr = yytext; return UNIT; // mega Hertz
+"GHz"                        COUNTCHARS; yylval.quantity.scaler  = Quantity::GigaHertz;           yylval.quantity.unitStr = yytext; return UNIT; // giga Hertz
+
```

Comment:
Terahertz as well? I think this scale is used in communications or something like that.

Reply:
> Terahertz as well? I think this scale is used in communications or something like that.

Indeed. I added it now too.